docs/UserGuide.md

```diff
@@ -157,39 +187,159 @@ Examples:
+* Only full words will be matched e.g. `Meeting` will not match `Meetings`.
+* Tasks matching at least one keyword will be returned e.g. `n/Meeting n/Shareholders` will return the persons `Engagement with Shareholders` and `Annual Meeting`.
+
+</div>
+
+### 2.3 Strategic Planning
+
+#### 2.3.1 Load new background image: `load-court`
+
+* (To be filled up by @abdul)
+
+#### 2.3.2 Clearing all entries: `clear-s`
+
+* (To be filled up by @xiaoteng)
+
+#### 2.3.2 Adding new players: `add-s`
+
+* (To be filled up by @xiaoteng)
+
+#### 2.3.3 Moving a player to a coordinate: `move`
+
+* (To be filled up by @xiaoteng)
+
+#### 2.3.4 Export strategy board as image file: `export`
+
+* (To be filled up by pragya)
+
+#### 2.3.5 Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
 
+
 --------------------------------------------------------------------------------------------------------------------
 
-## FAQ
+## **3. FAQ**
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Coach2k22 home folder.
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Command summary
-
-| Action       | Format, Examples                                                                                                   |
-|--------------|--------------------------------------------------------------------------------------------------------------------|
-| **List**     | `list`                                                                                                             |
-| **Add**      | `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​` <br> e.g., `add n/Johnson p/83918273 e/johnson@gmail.com t/Hustlers` |
-| **Del**      | `del INDEX`<br> e.g., `del 1`                                                                                      |
-| **Add Tag**  | `tag-add INDEX  TAG_NAME`<br> e.g., `tag-add 1 Public Relations`                                                   |
-| **Del Tag**  | `tag-del INDEX  TAG_INDEX`<br> e.g., `tag-del 1 1`                                                                 |
-| **Find**     | `find [n/NAME]…​ [t/TAG]…​`<br> e.g., `find n/Alex n/Charlotte t/team1 t/team2`                                                     |
-| **Add Note** | `note-add INDEX  NOTE_DESCRIPTION`<br> e.g., `note-add 1 Surgery Scheduled for tomorrow`                           |
-| **Del Note** | `note-del INDEX  NOTE_INDEX`<br> e.g., `note-del 1 1`                                                              |
-| **Help**     | `help`                                                                                                             |
-| **Exit**     | `exit`                                                                                                             |
+## **4. Command summary**
```

Comment:
I think the `help` command is missing? And I also think it is not good to put `exit` command under strategic planning section. Maybe can make a new section for this two like "General"

Reply:
Thanks for the suggestion. I don't think we have a help command in our codebase which is the reason for its removal. I will create another section named `General` but will it look weird since there's only one command for that? Additionally, I am actually planning to refactor the file `AddressBookParser` into `Coach2K22Parser` at the end of v1.3b since our app is no longer called `AddressBook`. The label `Coach2K22Parser` was used in anticipation for that

Reply:
Ok, if this is the case. Than sounds good to me. For General Commands, I think we should have 2, one is `help` and one is `exit`